image/v1/types.go

```diff
@@ -605,6 +605,10 @@ type ImageBlobReferences struct {
 	// not have separate config blobs and this field will be set to nil if so.
 	// +optional
 	Config *string `json:"config" protobuf:"bytes,2,opt,name=config"`
+	// manifests is the list of manifests that compose this image. For a
+	// single architecure image, it is empty. For a multi-arch image, it
```

Comment:
This seems weird to me.  Why doesn't a single architecture image contain a single manifest?

Reply:
The reason for manifest lists existing is for one image to support multiple architectures. A manifest list could in theory only list a single architecture, but when this is the case people use a single manifest (as opposed to a manifest list). Single manifests do not point to other manifests (instead of manifests they have layers and a config for example), thus should not use this field.

Does that help? Perhaps there's something we can improve in the comment 🤔

Reply:
indeed, I rephrased it make it more clear.